brave/browser-laptop
# Eth waterloo

Fixed #11532

Submitter Checklist:


Test Plan:

Case 1:

* ensure MetaMask is disabled via about:preferences#extensions
* visit https://tokenfactory.surge.sh
* ensure that you receive a "This page contains a Dapp, would you like to enable the MetaMask extension?" notification
* click on "Install MetaMask"
* ensure that MetaMask is installed (verify via about:preferences#extensions)
* once installed, refresh  visit https://tokenfactory.surge.sh
* ensure that you can see the "Block Numbers" being updated

Case 2:

* ensure MetaMask is disabled via about:preferences#extensions
* visit https://tokenfactory.surge.sh
* ensure that you receive a "This page contains a Dapp, would you like to enable the MetaMask extension?" notification
* click on "No Thanks"
* refresh the website and ensure that the notification isn't being displayed anymore

Case 3:

* ensure MetaMask is enabled via about:preferences#extensions
* visit https://tokenfactory.surge.sh
* ensure that the "DApp" notification isn't being displayed

Case 4:

* ensure MetaMask is disabled via about:preferences#extensions
* visit https://brianbondy.com/
* ensure that the "DApp" notification isn't being displayed

Reviewer Checklist:

Tests
